tutorials/plot_object_epochs.py

```diff
@@ -75,6 +75,23 @@
 print(epochs[1:5])
 print(epochs['Auditory/Right'])
 
+###############################################################################
+# Note the '/'s in the event code labels. These separators allow tag-based
+# selection of epoch sets; every string separated by '/' can be entered, and
+# returns the subset of epochs matching any of the strings. E.g.,
+
+print(epochs['Right'])
+print(epochs['Right', 'Left'])
+
+
+###############################################################################
+# Note that MNE will not complain if you ask for tags not present in the object,
+# as long as it can find some match.
```

Comment:
", as the ['Right', 'Left'] below is treated as "Right" or "Left", and the object does have "Right"-type epochs."

or something like this.

You could even put back the one that raises an error (`'Left'`), just put it in a try/except. It seems worth keeping to point out that if none can be found you do get an error.